config/jobs/kubernetes-sigs/sig-windows/soak-tests.yaml

```diff
@@ -1,62 +1,61 @@
 ---
+presets:
+- labels:
+    preset-azure-win-soak-test-creds: "true"
+  env:
+  - name: KUBECONFIG
+    value: "/root/.kube/win-soak-test"
+  volumeMounts:
+  - name: win-soak-test
+    mountPath: /root/.kube/win-soak-test
+  volumes:
+  - name: win-soak-test
+    secret:
+      secretName: win-soak-test
+      defaultMode: 420
+      items:
+      - key: win-soak-test
+        path: win-soak-test.kubeconfig
 presubmits:
-  kubernetes/perf-tests:
+  kubernetes/test-infra:
   - name: soak-tests-capz-windows-2019
     decorate: true
     always_run: false
     optional: true
     decoration_config:
       timeout: 8h
     path_alias: k8s.io/perf-tests
+    cluster: win-soak-test
```

Comment:
What does this value do?

Reply:
This changes where the prow job is scheduled to run, overwriting the default kubeconfig context.

Reply:
I think we need to remove this because we are storing the secret in Azure and not using https://github.com/kubernetes/test-infra/blob/master/prow/create-build-cluster.sh to build a trusted GKE cluster